packages/config/src/options.js

```diff
@@ -43,6 +43,21 @@ export function getNuxtConfig(_options) {
     options.extensions = [options.extensions]
   }
 
+  const styleExtensionDefaults = ['css', 'pcss', 'postcss', 'styl', 'stylus', 'scss', 'sass', 'less']
```

Comment:
If defaults ordering would be important I suggest mentioning it in the docs too :)

Reply:
Yeah, I'll create a doc PR afterwards anyway :relaxed: 
Thanks for the heads up!